compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1919,3 +1919,6 @@ error.unsupported.computed.resource.access.path.segment.type=\
 error.unsupported.resource.access.rest.segment.type=\
     unsupported resource access rest segment type: expected array of ''int'', ''string'', ''float'', ''boolean'',\
    ''decimal'' but found ''{0}''
+
+error.int.range.overflow=\
+  'int' range overflow
```

Comment:
If we mention the supported limits, it will be useful for the user

Reply:
Updated 91289db7c7c17982b7483294300408a9fd1bb23b